lang/English/modules/settings.php

```diff
@@ -45,7 +45,7 @@
 define('feed_url', "Feed URL");
 define('feed_url_info', "GrayCube.com is sharing a LGSL feed on the URL:<br><b>http://www.greycube.co.uk/lgsl/feed/lgsl_files/lgsl_feed.php</b>");
 define('charset', "Character Encoding");
-define('charset_info', "UTF8, ISO, ASCII, etc... Left it blank to use ISO encoding.");
+define('charset_info', "UTF8, ISO, ASCII, etc... Leave it blank to use ISO encoding.");
```

Comment:
I'm not sure this string now reflects what it does. I think if it is empty, it uses what in defined in the lang file (which is now UTF-8 for all because of transifex).